waspc/src/Wasp/Psl/Db.hs

```diff
@@ -0,0 +1,10 @@
+module Wasp.Psl.Db
+  ( dbToPrismaProviderId,
+  )
+where
+
+import qualified Wasp.AppSpec.App.Db as AS.Db
```

Comment:
This is also indicator that something is a bit weird, importing AppSpec in Wasp.Psl module tree. We kind of want Wasp.Psl to be a standalone concept, ideally not reliant on the rest of Wasp's sepcifics, if possible. So if somebody had to rely on somebody, it would probably be better to have AppSpec rely on Prisma, than the opposite direction.